src/include/snapper/dialogs.rb

```diff
@@ -309,12 +317,12 @@ def CreateSnapshotPopup(pre_snapshots)
       UI.ChangeWidget(
         Id("post"),
         :Enabled,
-        Ops.greater_than(Builtins.size(pre_items), 0)
+	!pre_items.empty?
```

Comment:
I will send another PR rewriting YCP-to-Ruby-autoconversion code but its not related to the bug so i prefer to send a separated one.